### PR TITLE
Changes to support assigning taxonomy to ASV using MAPseq output

### PR DESCRIPTION
**Pull request by chrisAta**
Recently changed the tool used in V6 amplicon for assigning taxonomy to ASVs from DADA2 to MAPseq. This change has necessitated adding a conversion script (mapseq2asvtable.py) and updating the make_asv_count_table.py script.

Also ran prettier for the first time on this repo so quite a few small changes were made by it

### CODE REVIEW

File: mgnify_pipelines_toolkit/analysis/amplicon/make_asv_count_table.py [hunk 1]
```diff
@@ -83,47 +85,53 @@ def make_tax_assignment_dict_silva(taxa_df, asv_dict):
 
         while True:
 
+            if sk != "0":
+                sk = "_".join(sk.split(" "))
+                tax_assignment += f"{sk}"
```
Comment (mberacochea) [inline mgnify_pipelines_toolkit/analysis/amplicon/make_asv_count_table.py:90]:
```suggestion
                tax_assignment += sk
```

File: mgnify_pipelines_toolkit/analysis/amplicon/make_asv_count_table.py [hunk 2]
```diff
@@ -161,45 +170,55 @@ def make_tax_assignment_dict_pr2(taxa_df, asv_dict):
 
             if d != "0":
                 d = "_".join(d.split(" "))
-                tax_assignment += f"d__{d}"
+                tax_assignment += f"{d}"
```
Comment (mberacochea) [inline mgnify_pipelines_toolkit/analysis/amplicon/make_asv_count_table.py:173]:
```suggestion
                tax_assignment += d
```

**Review by mberacochea — CHANGES_REQUESTED** [review]
It looks good to me, I didn't do an extensive review but the code looks alright to me.
I added some minor comments

**Comment by chrisAta** [comment]
> It looks good to me, I didn't do an extensive review but the code looks alright to me. I added some minor comments

thank you! changes done

**Review by mberacochea — APPROVED** [review]
:shipit:

**Pull request merged**